pom.xml

```diff
@@ -16,7 +16,7 @@
 
     <!-- Project-specific properties -->
     <exec.mainClass>com.google.sps.ServerMain</exec.mainClass>
-    <googleCloudProjectId>YOUR_PROJECT_ID_HERE</googleCloudProjectId>
+    <googleCloudProjectId>cgordillo-sps-spring21</googleCloudProjectId>
```

Comment:
I think this needs to be the team id, something like `spring21-sps-team41` - technical chat says it can be found at http://console.cloud.google.com